File: backend/src/main/java/by/kapitonov/cinema/backend/service/dto/hall/UpdateHallDTO.java
package by.kapitonov.cinema.backend.service.dto.hall;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode
public class UpdateHallDTO {

    private Long id;
    private String hallName;
    private Integer floor;
    private Integer rowsNumbers;
    private Integer numberSeatsPerRow;
    private String status;

}

File: frontend/src/app/model/dto/hall/UpdateHallDTO.ts
import {Hall} from "../../Hall";

export class UpdateHallDTO {

  id?: number;
  hallName?: string;
  floor?: number;
  rowsNumbers?: number;
  numberSeatsPerRow?: number;
  status: string;

  public constructor(hall: Hall) {
    this.id = hall.id;
    this.hallName = hall.hallName;
    this.floor = hall.floor;
    this.rowsNumbers = hall.rowsNumbers;
    this.numberSeatsPerRow = hall.numberSeatsPerRow;
    this.status = hall.status;
  }

}
